lib/ansible/modules/database/aerospike/aerospike_migrations.py

```diff
@@ -0,0 +1,562 @@
+            - if something else is changing the cluster, we may want to fail
+        required: False
+        type: bool
+        default: True
+    migrate_tx_key:
+        description:
+            - The metric key used to determine if we have tx migrations
+            - remaining. Changeable due to backwards compatibility.
+        required: False
+        type: str
+        default: migrate_tx_partitions_remaining
+    migrate_rx_key:
+        description:
+            - The metric key used to determine if we have rx migrations
+            - remaining. Changeable due to backwards compatibility.
+        required: False
+        type: str
+        default: migrate_rx_partitions_remaining
+    target_cluster_size:
+        description:
+            - When all aerospike builds in the cluster are greater than
+            - version 4.3, then the 'cluster-stable' info command will be used.
+            - Inside this command, you can optionally specify what the target
+            - cluster size is - but it is not necessary. You can still rely on
+            - min_cluster_size if you don't want to use this option.
+            - If this option is specified on a cluster that has at least 1
+            - host <4.3 then it will be ignored until the min version reaches
+            - 4.3.
+        required: False
+        type: int
+'''
+EXAMPLES = '''
+# check for migrations on local node
+- name: wait for migrations on local node before proceeding
+  aerospike_migrations:
+    host: "localhost"
+    connect_timeout: 2000
+    consecutive_good_checks: 5
+    sleep_between_checks: 15
+    tries_limit: 600
+    local_only: False
+
+# example playbook:
+---
+- name: upgrade aerospike
+  hosts: all
+  become: true
+  serial: 1
+  tasks:
+    - name: Install dependencies
+      apt:
+        name:
+            - python
+            - python-pip
+            - python-setuptools
+        state: latest
+    - name: setup aerospike
+      pip:
+          name: aerospike
+# check for migrations every (sleep_between_checks)
+# If at least (consecutive_good_checks) checks come back OK in a row, then return OK.
+# Will exit if any exception, which can be caused by bad nodes,
+# nodes not returning data, or other reasons.
+# Maximum runtime before giving up in this case will be:
+# Tries Limit * Sleep Between Checks * delay * retries
+    - name: wait for aerospike migrations
+      aerospike_migrations:
+          local_only: True
+          sleep_between_checks: 1
+          tries_limit: 5
+          consecutive_good_checks: 3
+          fail_on_cluster_change: true
+          min_cluster_size: 3
+          target_cluster_size: 4
+      register: migrations_check
+      until: migrations_check is succeeded
+      changed_when: false
+      delay: 60
+      retries: 120
+    - name: another thing
+      shell: |
+          echo foo
+    - name: reboot
+      reboot:
+'''
+
+RETURN = '''
+# im not real sure what goes here
```

Comment:
In case your module returns some data (like information returned by Aerospike), you can document that here. If there isn't anything, you can simply leave this empty (or keep a comment like this :) ).

Reply:
It will only return a boolean. The function doesn't change anything.. If false it should return a reason. So I guess I will notate that as a comment?

Reply:
If it's just the boolean `changed`, you can leave it empty.